lib/supplierlandingpage/constants.dart

```diff
@@ -0,0 +1,28 @@
+const titleFontSize = 38.0;
```

Comment:
Why do you have your own constants file? It should be just 1 single file for the entire project, unless it contains only constants that are really necessary just for your widget, but I doubt it, since here you have things related to register. Is this a mistaken duplicate of the original constants file?